app/Nova/AttendanceExport.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova;
+
+use App\Nova\Fields\Hidden;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Panel;
+
+class AttendanceExport extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = \App\AttendanceExport::class;
+
+    /**
+     * The relationships that should be eager loaded on index queries.
+     *
+     * @var array<string>
+     */
+    public static $with = ['downloadUser'];
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'name';
```

Comment:
This column doesn't exist. Maybe use `id` instead?

```suggestion
    public static $title = 'id';
```